cli/cmd/register_test.go

```diff
@@ -141,7 +141,7 @@ func testDeployCfg(t *testing.T) deployConfig {
 
 	return deployConfig{
 		deployer:         eoa.MustAddress(netconf.Devnet, eoa.RoleDeployer),
-		owner:            eoa.MustAddress(netconf.Devnet, eoa.RoleAdmin),
+		manager:          eoa.MustAddress(netconf.Devnet, eoa.RoleManager),
```

Comment:
doesn't seem like we configure the `upgrader` in the contracts we deploy? So I don't think it will be able to upgrade anything....

Reply:
you are right, this should be the upgrader because it gets bound to the proxy contract, fixed: 72555a2241dfd86df0aab1f6e5439c71dcc46d37